subprojects/packagefiles/avro-cpp/meson_options.txt

```diff
@@ -0,0 +1,6 @@
+option(
+    'build_executable',
+    type: 'boolean',
```

Comment:
FYI on other Apache projects we have been using `feature` options instead of boolean. I don't think its a huge deal, but probably nice to stick with that pattern for consistency in the ecosystem